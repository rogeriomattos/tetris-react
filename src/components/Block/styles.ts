import styled from 'styled-components';

export const BlockContainer = styled.div`
    display: block;
    width: 30px;
    height: 30px;
    background: transparent;
    border-color: #000000;
    border-width: 2px;
    border-style: solid;
    box-sizing: border-box;
    position: relative;
    opacity: 0.1;
    border-radius: 2px;
    &:after {
        content: "";
        position: absolute;
        background: #000;
        width: calc(100% - 4px);
        height: calc(100% - 4px);
        transform: translate(-50%, -50%);
        top: 50%;
        left: 50%;
        border-radius: 2px;
    }
`;