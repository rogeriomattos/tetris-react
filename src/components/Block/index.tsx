import { BlockContainer } from './styles';

const Block = () => (
    <BlockContainer data-testid="block"/>
);

export default Block;