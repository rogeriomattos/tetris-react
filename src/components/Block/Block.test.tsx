import { render } from '@testing-library/react';
import Block from '.';

describe('components/Block', () => {
    it('Should render correctly', () => {
        const { container } = render(<Block/>);
        expect(container).toMatchSnapshot();
    })
});