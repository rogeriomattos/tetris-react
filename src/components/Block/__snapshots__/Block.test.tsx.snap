// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/Block Should render correctly 1`] = `
<div>
  <div
    class="sc-bcXHqe jUEyRL"
    data-testid="block"
  />
</div>
`;
